spec/string-indices.js

```diff
@@ -0,0 +1,22 @@
+import chai, { expect} from 'chai'
+import chaiChange from 'chai-change'
+import StringIndices from '../src/string-indices'
+
+chai.use(chaiChange)
+
+describe('StringIndices', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(StringIndices).to.be.a('function')
+  })
+
+  it('returns the string in the sentence found at the index', () => {
+    expect(StringIndices('catch me outside how about that', 1)).to.equal('catch')
+  })
+
+  it('should return an empty string if you ask for the word at index 7', () => {
```

Comment:
I would be more specific to what you are testing for. You are not testing to make sure you only care about words 1-6 of any string. This is meant to test for asking for an index that does not exist. I would write this as `returns empty string if the index requested does not exist`